vllm/worker/model_runner.py

```diff
@@ -1335,6 +1341,13 @@ def execute_model(
             logits=logits,
             sampling_metadata=model_input.sampling_metadata,
         )
+        if self.observability_config.collect_model_forward_time:
+            torch.cuda.synchronize()
```

Comment:
it looks like there's a `Event.synchronize()` which seems a better fit than this global synchronize() https://pytorch.org/docs/stable/generated/torch.cuda.Event.html

Reply:
Done. Thanks for the tip!